### PR TITLE
CCS changes for Tucson test-stand

### PR DESCRIPTION
**Pull request by glennmorris**
Changes related to Tucson test-stand. The only change that applies elsewhere is the one to hieradata/cluster/comcam-ccs.yaml, but it should be a no-op.

These changes should address most of the issues from
https://jira.lsstcorp.org/browse/IT-3277?jql=project%20%3D%20IT%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20%3D%20tucson-test-stand%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC

### CODE REVIEW

**Comment by glennmorris** [comment]
Force pushed to fix a last minute typo, will close and reopen for clarity.

**Comment by glennmorris** [comment]
Seems github doesn't let me replace this pull request, so I will reopen it.
Sorry for making even more noise!

File: hieradata/site/tu/cluster/comcam-ccs.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+ccs_site: "tucson"
```
Comment (jhoblitt) [inline hieradata/site/tu/cluster/comcam-ccs.yaml:2]: :+1:

File: hieradata/site/tu/cluster/comcam-ccs.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+## FIXME this should be "comcam-mcm-dds", same as real comcam.
+ccs_sal::dds_interface: "140.252.147.52"
```
Comment (jhoblitt) [inline hieradata/site/tu/cluster/comcam-ccs.yaml:5]: The `dds` interface still hasn't shown up in foreman for some reason.  I'll sort this out tomorrow.

File: site/profile/manifests/ccs/daq_client.pp [hunk 3]
```diff
@@ -0,0 +1,77 @@
+## @summary
+##   Installation of DAQ client:
+##   Download DAQ tarfile, extract it, create /etc/ccs config files.
+
+class profile::ccs::daq_client (
```
Comment (jhoblitt) [inline site/profile/manifests/ccs/daq_client.pp:5]: This appears to be completely stand alone and could be in an external module.  There are a bunch of example of things in the tree that should be in an external module as they aren't a composition.  I don't object to merging this now but it should probably be moved external at some point in the future in the interest of keeping our profile tree small.

File: hieradata/org/lsst/role/ccs-mcm.yaml [hunk 4]
```diff
@@ -3,6 +3,7 @@ classes:
   - "profile::core::common"
   - "profile::ccs::common"
   - "profile::ccs::tomcat"
+  - "profile::ccs::graphical"
```
Comment (jhoblitt) [inline hieradata/org/lsst/role/ccs-mcm.yaml:6]: FYI - I usually `sort` the inclusion list but the linter doesn't care.

**Review by jhoblitt — APPROVED** [review]
@glennmorris Do you want me to merge this and move the `tu_production` branch forward?

File: site/profile/manifests/ccs/daq_client.pp (hunk 3 above)
Comment (glennmorris) [reply site/profile/manifests/ccs/daq_client.pp:5]: Thanks, I will try to separate it out at some point in the future.

File: hieradata/org/lsst/role/ccs-mcm.yaml (hunk 4 above)
Comment (glennmorris) [reply hieradata/org/lsst/role/ccs-mcm.yaml:6]: Noted.

**Pull request merged**